tests/draft4/ref.json

```diff
@@ -141,6 +141,59 @@
         ]
     },
     {
+        "description": "recursive refs",
+        "schema": {
+            "definitions": {
+                "a": {"$ref": "#/definitions/b"},
+                "b": {"$ref": "#/definitions/a"}
+            },
+            "properties": {
+                "foo": {
+                    "type": "integer"
+                },
+                "bar": {
+                    "$ref": "#/definitions/a"
+                }
+            }
+        },
+        "tests": [
+            {
+                "description": "unrelated property",
+                "data": {"foo": 1},
+                "valid": true
+            }
+        ]
+    },
+    {
```

Comment:
I've removed the recursive tests now. It's good to know that the draft spec has been updated to include that.